shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -95,6 +96,14 @@ protected ConsistencyLevel getConsistency(CommandLine cl) {
     }
   }
 
+  protected String getUserData(CommandLine cl) {
+    if (cl.hasOption(userDataOpt.getOpt())) {
+      return cl.getOptionValue(userDataOpt.getOpt());
+    } else {
+      return "Accumulo Shell";
```

Comment:
The shell could have a unique ID it uses by default here, so individual shell instances can be distinguished from one another.

Reply:
ScanCommand now uses:
```
"shell-" + shellState.getAccumuloClient().whoami() + "-" + getCorrelationId(cl);
```